client/index.html

```diff
@@ -25,7 +25,7 @@
 	<meta name="theme-color" content="#455164">
 
 	</head>
-	<body class="signed-out <%= public ? "public" : "" %>">
+	<body class="signed-out {{ternary public 'public' '' }}">
```

Comment:
:man_facepalming:
I mean, it feels so awkward to have to roll-your-own ternary operator. I get the minimalist approach, but still frustrating.

Looking at all uses of `ternary`, I think we should go for native `{{#if public}}public{{/if}}`, especially since all have an empty string in their `else` block.

Kind of a nitpick, sorry, but I think we should embrace the Handlebars way instead of defining our programming language 😕.